website/docs/r/project_instance.html.markdown

```diff
@@ -0,0 +1,159 @@
+* `name` - (Required, String) The project name.
+  * Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `/^(?!\\s).+\\S$/`.
+* `resource_group` - (Optional, String) Group name of the customized collection of resources.
+  * Constraints: The default value is `Default`. The maximum length is `40` characters. The minimum length is `0` characters. The value must match regular expression `/^$|^(?!\\s).*\\S$/`.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your resource is created.
+
+* `id` - The unique identifier of the Project definition.
+* `crn` - (String) An IBM Cloud resource name, which uniquely identifies a resource.
+  * Constraints: The maximum length is `512` characters. The minimum length is `9` characters. The value must match regular expression `/^crn:v[0-9](:([A-Za-z0-9\\-._~!$&'()*+,;=@\/]|%[0-9A-Z]{2})*){8}$/`.
+* `metadata` - (List) The metadata of the project.
+Nested scheme for **metadata**:
+	* `created_at` - (String) A date/time value in the format YYYY-MM-DDTHH:mm:ssZ or YYYY-MM-DDTHH:mm:ss.sssZ, matching the date-time format as specified by RFC 3339.
+	* `crn` - (String) An IBM Cloud resource name, which uniquely identifies a resource.
+	  * Constraints: The maximum length is `512` characters. The minimum length is `9` characters. The value must match regular expression `/^crn:v[0-9](:([A-Za-z0-9\\-._~!$&'()*+,;=@\/]|%[0-9A-Z]{2})*){8}$/`.
+	* `cumulative_needs_attention_view` - (List) The cumulative list of needs attention items for a project.
+	  * Constraints: The maximum length is `10000` items. The minimum length is `0` items.
+	Nested scheme for **cumulative_needs_attention_view**:
+		* `config_id` - (String) The unique ID of a project.
+		  * Constraints: The maximum length is `128` characters. The value must match regular expression `/^[\\.\\-0-9a-zA-Z]+$/`.
+		* `config_version` - (Integer) The version number of the configuration.
+		* `event` - (String) The event name.
+		* `event_id` - (String) The unique ID of a project.
+		  * Constraints: The maximum length is `128` characters. The value must match regular expression `/^[\\.\\-0-9a-zA-Z]+$/`.
+	* `cumulative_needs_attention_view_err` - (String) \"True\" indicates that the fetch of the needs attention items failed.
+	* `event_notifications_crn` - (String) The CRN of the event notifications instance if one is connected to this project.
+	* `location` - (String) The location where the project was created.
+	* `resource_group` - (String) The resource group where the project was created.
+	* `state` - (String) The project status value.
+	  * Constraints: The maximum length is `64` characters. The minimum length is `1` character. The value must match regular expression `/^(CREATING|CREATING_FAILED|UPDATING|UPDATING_FAILED|READY)$/`.
+
+## Provider Configuration
```

Comment:
Remove these sections of docs from this resource doc..
```
## Provider Configuration
### Static credentials
### Environment variables
```

Reply:
done